CONTRIBUTING.md

```
@@ -62,7 +62,7 @@ If you’d like to use the repository from source, you can either install from g
 To install via git:
 
 ```sh
-$ pip install git+ssh://git@github.com/businessradar/businessradar-sdk-python#master.git
+$ pip install git+ssh://git@github.com/businessradar/businessradar-sdk-python.git
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)